doc/source/features/pxc-strict-mode.rst

```diff
@@ -343,8 +343,16 @@ the following happens:
 Combining schema and data changes in a single statement
 -------------------------------------------------------
 
-|PXC| does not support ``CREATE TABLE ... AS SELECT`` (CTAS) statements,
-because they combine both schema and data changes.
+Starting from version 8.0, |PXC| supports the :abbr:`CTAS (CREATE TABLE ... AS
```

Comment:
There is no change in this stand. We officially continue to say that under ENFORCING mode we don't support CTAS. But complete disable is not good choice so we kept these feature enabled under DISABLED or permissive mode.

Reply:
May be we can re-word the statement to say PXC doesn;'t recommend using CTAS as it is mix of DDL and DML changes and with gtid_on MySQL too report it as error as it can affect GTID consistency. Given the said limitation, PXC has blocked use of CTAS under ENFORCING and user should avoid using CTAS.